Example with default values for props (#1208)
withastro/docs: src/pages/en/core-concepts/astro-components.md

<h2>{greeting}, {name}!</h2>
```

> Component props can be given default values to use when the consumer does not provide them.

Reviewer: ```suggestion
Component props can be given default values to use when none are provided.
```
I think I might prefer something shorter like this? What do you think?
